rebasebot/bot.py

```diff
@@ -427,13 +430,13 @@ def _init_working_dir(
     return gitwd
 
 
-def _manual_rebase_in_repo(repo: github3.github.repo.Repository) -> bool:
+def _manual_rebase_in_repo(repo: github3.github.repo.Repository):
```

Comment:
Nit: You could rewrite the signature to return `Optional(github3.pulls.PullRequest)` and check the result for None then.